tests/test_field_validation.py

```
@@ -0,0 +1,440 @@
+        assert len(rule_set.rules) == 5  # email_format, email_not_null, age_range, name_length, status_enum
+
+
+class TestTableFieldValidator:
+    """Test table-level field validation."""
+    
+    def test_table_field_validator_initialization(self, temp_config_with_db):
+        """Test TableFieldValidator initialization."""
+        config = get_config(temp_config_with_db)
+        connection_manager = get_connection_manager(config)
+        validator = TableFieldValidator(connection_manager)
+        assert validator.connection_manager is not None
```

Ruff (0.12.2)

Use of `assert` detected

(S101)